spec/features/admin/merchants/new_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe 'The merchants index page,' do
+
+  before :all do
+    @merchant_1 = FactoryBot.create(:merchant)
+  end
+
+  describe 'Create new merchant as Admin,' do
+    it 'loads form to create a new merchant' do
+      visit new_admin_merchant_path
+
+      within '#form' do
+        fill_in('merchant_name', with: 'Fancy Petz')
```

Comment:
This works too but I'd recommend using `merchant[name]` instead as it's the ID and not the name of the field.